roles/openshift_logging_elasticsearch/defaults/main.yml

```diff
@@ -38,7 +36,11 @@ openshift_logging_elasticsearch_node_topology:
     requests:
       cpu: 100m
       memory: 256Mi
-
+    pv_selector:
+      label: value
+    pvc_size: 10Gi
+    pvc_name: pvc1
+    storage_group: 6566
```

Comment:
where does `6566` come from?

Reply:
that was a type. added the correct value 65534